test/conformance/ingress/util.go

```diff
@@ -985,7 +986,8 @@ func CreateTLSSecret(ctx context.Context, t *testing.T, clients *test.Clients, h
 			Name:      name,
 			Namespace: test.ServingNamespace,
 			Labels: map[string]string{
-				"test-secret": name,
+				"test-secret":              name,
+				pkg.SecretInformerLabelKey: "test",
```

Comment:
This is dummy we only need this because of the fact that net-istio uses the tests in this repo.